Motorcycle.h

```diff
@@ -1,13 +1,18 @@
 #pragma once
 
-#include <iostream>
-#include <limits>
+//#include <iostream>
+//#include <limits>
+#include "Vehicle.h"
 
-struct Motorcycle : public Vehicle
+struct Motorcycle : Vehicle
 {
     Motorcycle(const std::string& n);
+
+    virtual ~Motorcycle() override;
+    Motorcycle(const Motorcycle&);
+    Motorcycle& operator=(const Motorcycle&) = default;
```

Comment:
move this copy-assign operator implementation to the cpp file, with the other implementations.
- [ ] revise